### PR TITLE
upgrade gflags version for rhel8

### PR DESCRIPTION
**Pull request by dtrawins**
fix seg fault in cpp sample apps on rhel8

### CODE REVIEW

**Review by p-durandin — CHANGES_REQUESTED** [review]
Should we use platform spec or "$arch@ here

File: scripts/install_dependencies/install_openvino_dependencies.sh [hunk 1]
```diff
@@ -228,11 +228,11 @@ elif [ "$os" == "centos7" ] || [ "$os" == "centos8" ] ||
         pkgs_core+=(
             https://vault.centos.org/centos/8/AppStream/$arch/os/Packages/tbb-2018.2-9.el8.$arch.rpm
             https://download-ib01.fedoraproject.org/pub/epel/8/Everything/$arch/Packages/p/pugixml-1.13-1.el8.$arch.rpm
-            https://vault.centos.org/centos/8/PowerTools/$arch/os/Packages/gflags-2.1.2-6.el8.$arch.rpm
+	    http://mirror.centos.org/centos/8-stream/PowerTools/$arch/os/Packages/gflags-2.2.2-1.el8.$arch.rpm
```
Comment (ilyachur) [inline scripts/install_dependencies/install_openvino_dependencies.sh:231]: Please fix the indentation.
```suggestion
            http://mirror.centos.org/centos/8-stream/PowerTools/$arch/os/Packages/gflags-2.2.2-1.el8.$arch.rpm
```

**Review by ilyachur — COMMENTED** [review]
In general LGTM

**Comment by ilya-lavrenov** [comment]
Solution is here https://github.com/openvinotoolkit/openvino/pull/14724

**Pull request closed without merging**